extensions/vault/runtime/src/main/java/io/quarkus/vault/internal/client/dto/AbstractVaultDTO.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.vault.internal.client.dto;
+
+public class AbstractVaultDTO<DATA, AUTH> implements VaultModel {
+
+    public String request_id;
+    public String lease_id;
+    public boolean renewable;
+    public int lease_duration;
+    public DATA data;
+    public Object wrap_info;
```

Comment:
Same here for the naming conventions. No underscore please. Hum, except if it's needed by something specific? I see it's a model, maybe it needs to be like that for deserialization purposes?

Reply:
done https://github.com/quarkusio/quarkus/pull/2897#discussion_r326829173

> maybe it needs to be like that for deserialization purposes?

yes, it was